stephenPrintup/README.md

```diff
@@ -0,0 +1,276 @@
+### Homework #1
+(Due Monday 7/20)
+
+You need to turn in this homework by 
+
+[X]cloning this repo, 
+[X]making a new branch with your solutions, 
+[X]pushing to github, and 
+[X]issuing a pull request.
+
+For details, follow the instructions [here](http://portlandcodeschool.github.io/jse/2015/01/07/command-line-and-git-slides/#/14).
+
+Please also read the [learning objectives](objectives.md) for this week.
+
+---
+
+**1)** (_30% of total time_)
+Predict the output of each of the following expressions, and explain the rules which determine that answer.  (_Hint: The rules for operator `==` are complicated, but do your best!_)
+
+If the output depends on the value or type of variable _x_, identify the conditions (what _x_ is) when the expression will output true (or false, if that's simpler).  Assume the cases are independent, and _x_ is reset to an unknown value before each.
+
+Some of these are tricky!  Don't trust your first instinct.  
+
+
+**a)** `"1" == 1`
+
+prediction 	11							<---wrong
+result 		true
+
+**b)** `"1" === 1`
+
+prediction	false
+result      false
+
+**c)** `x == 'x'`
+
+prediction true								<----wrong
+result     undefined
+
+**d)** `x == (x+'')`
+
+prediction undefined
+result     undefined
+
+**e)** `'' == ' '`
+
+prediction  true							<----wrong
+result      false
+
+**f)** `x = true`
+
+prediction  true
+result      true
+
+**g)** `var x; x == 'undefined'`
+
+prediction  undefined							<----wrong
+result      false
+
+**h)** `'9'<'10'`
+
+prediction true
+result     false 				            	<---reveiw
+
+**i)** `typeof x + 1 === "number"`
+
+prediction false, because "number" is a string
+result     false
+
```

Comment:
Not entirely the correct reasoning. Because the evaluation of 'typeof x + 1' returns "number1", it is not the fact that "number" is a string that is the issue, rather that "number1" != "number". Remember that the first thing that occurs here is that typeof evaluates 'x' (which, let's say is = 1), and returns "number". Javascript then adds "1" to that and evaluates against "number" on the right side, returning false.